app/controllers/dashboard/profile_controller.rb

```diff
@@ -13,10 +13,28 @@ def update
     end
   end
 
+  def invite_friends
+    @incorrect_emails = FriendsService.invite(invite_friends_emails)
+    if @incorrect_emails.blank?
+      flash.now[:notice] = 'Ваши друзья успешно приглашены на сайт.'
+    else
+      flash.now[:alert] = 'Проверьте формат вводимых данных: ' + @incorrect_emails + '.'
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.